src/app/config/groups/constants.js

```diff
@@ -0,0 +1,7 @@
+export const LOAD_GROUP_FAILURE = "LOAD_GROUP_FAILURE";
+export const LOAD_GROUP_SUCCESS = "LOAD_GROUP_SUCCESS";
+export const LOAD_GROUP_PROGRESS = "LOAD_GROUP_PROGRESS";
+
+export const UPDATE_GROUP_FAILURE = "UPDATE_GROUP_FAILURE";
+export const UPDATE_GROUP_SUCCESS = "UPDATE_GROUP_SUCCESS";
+export const UPDATE_GROUP_PROGRESS = "UPDATE_GROUP_PROGRESS";
```

Comment:
Just a suggestion for next time, you can use a pseudo enum to batch up groups of actions types:

```
const GROUPS = Object.freeze({
     LOAD_GROUP_FAILURE: "LOAD_GROUP_FAILURE",
    ....
});
```